src/frontend/src/handler/create_sink.rs

```diff
@@ -746,6 +747,11 @@ static CONNECTORS_COMPATIBLE_FORMATS: LazyLock<HashMap<String, HashMap<Format, V
                     Format::Upsert => vec![Encode::Json],
                     Format::Debezium => vec![Encode::Json],
                 ),
+                MqttSink::SINK_NAME => hashmap!(
+                    Format::Plain => vec![Encode::Json,Encode::Bytes],
+                    Format::Upsert => vec![Encode::Json,Encode::Bytes],
+                    Format::Debezium => vec![Encode::Json,Encode::Bytes],
```

Comment:
We have not supported sinking raw bytes in any connector yet.

Reply:
I removed the bytes options, it's only working for sources